docs/source/plotting/ColorfillPlotsHelp.rst

```diff
@@ -1,5 +1,225 @@
 .. _Colorfill_Plots:
 
-===============
+===========================
+Colorfill and Contour Plots
+===========================
+
+.. TO UPDATE find these images in a .pptx file at https://github.com/mantidproject/documents/blob/master/Images/Images_for_Docs/formatting_plots.pptx
+
+**Other Plot Types**
+
+* :ref:`Basic_1D_Plots`
+* :ref:`Waterfall_Plots`
+* :ref:`3D_Plots`
+
+**General Plot Help**
+
+* :ref:`06_formatting_plots`
+* :ref:`02_scripting_plots`
+
+|
+|
+
+.. contents:: Colorfill and Contour Plots - Table of contents
+    :local:
+
 Colorfill Plots
-===============
\ No newline at end of file
+===============
+
+|
+|
+
+Plot Toolbar
+------------
+
+.. figure:: /images/PlotToolbarTiledColorfill.png
+   :alt: Plot Toolbar Colorfill
+   :align: center
+
+|
+|
+
+Click Menus
+-----------
+
+.. figure:: /images/PlotClickMenusColorfill.png
+   :alt: Click Menus Colorfill
+   :align: center
+   :width: 1500px
+
+|
+|
+
+|FigureOptionsGear.png| ptions Menu
+-----------------------------------
+
+.. figure:: /images/PlotOptionsColorfillContour.png
+   :alt: Plot Options Colorfill and Contour
+   :align: center
+
+
+|
+|
+
+
+Scripting
+---------
+
+Click the generate a script button |GenerateAScript.png| on a `Colorfill Plot <https://matplotlib.org/3.2.1/api/_as_gen/matplotlib.pyplot.imshow.html>`_:
+
+.. plot::
+   :include-source:
+
+   from mantid.simpleapi import *
+   import matplotlib.pyplot as plt
+   from matplotlib.colors import LogNorm
+
+   data = Load('MAR11060')
+
+   fig, axes = plt.subplots(subplot_kw={'projection':'mantid'})
+
+   # IMPORTANT to set origin to lower
+   c = axes.imshow(data, origin = 'lower', cmap='viridis', aspect='auto', norm=LogNorm())
+   axes.set_title('MAR11060')
+   cbar=fig.colorbar(c)
+   cbar.set_label('Counts ($\mu s$)$^{-1}$') #add text to colorbar
+   plt.show()
+
+For more advice: 
+
+.. * :ref:`01_basic_plot_scripting`
+
+* `Basic 1D 2D and 3D Plots <https://docs.mantidproject.org/tutorials/python_in_mantid/plotting/01_basic_plot_scripting.html>`_
```

Comment:
For me this link opens a 404 not found page.

Reply:
So I hardcoded the link to a page in the PythonInMantid Course, which is in a separate PR. This was lazy so I've reverted to the Ref link to `01_basic_plot_scripting` and this PR won't work until after my Python In Mantid PR is merged!